pkg/storer/internal/reserve/reserve.go

```diff
@@ -165,6 +165,7 @@ func (r *Reserve) Put(ctx context.Context, chunk swarm.Chunk) error {
 					if err != nil {
 						return fmt.Errorf("failed removing older chunk %s: %w", oldStampIndex.ChunkAddress, err)
 					}
+					r.size.Add(-1)
```

Comment:
this cannot be here, it's has to go outside the transaction.
the `r.size.Add(1)` is wrongly placed in the transaction. pls fix both.